components/phpunit_bridge.rst

```diff
@@ -356,6 +356,8 @@ In order to avoid making a real network connection, add the ``@dns-sensitive``
 annotation to the class and use the ``DnsMock::withMockedHosts()`` to configure
 the data you expect to get for the given hosts::
 
+    namespace Symfony\Component\Validator\Tests\Constraints;
```

Comment:
This is really strange. We show an example of your own app (`MyTest`) but then use an internal Symfony namespace. I don't understand this well, but I'd say this is not correct.